### DISCUSSION TITLE
[With Backend/refactor] swagger 사용과 관련된 논의

### DISCUSSION THREAD

**Discussion by locodingve** [slamapp/server#25] [General]
swagger 활용에 대해 논의하고자 합니다. 

🤔 논의 1. 주된 api 명세서는 swagger? restdocs? 
🤔 논의 2. 만약 restdocs를 주된 api명세서로 제공한다면, restdocs 완성되기 전에 swagger 자세하게 작성하여 제공 방향으로 진행?
🤔 논의 3. 언제까지 swagger 구체화 작업을 마무리 할까요~? 

관련 이슈 : [issue#23](https://github.com/slamapp/server/projects/1#card-78075914)

**Comment by KwonYeKyeong** [comment]
빠른 개발을 위해 현재로써는 swagger를 사용해야 할 것 같습니다.
swagger 작업 시 참고할만한 블로그 글을 첨부하겠습니다. [참고](https://ksshlee.github.io/spring/springboot/swagger/)
